tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py35,py27,docs
+skip_missing_interpreters = True
+
+[testenv]
+
+passenv = TRAVIS
+commands =
+    python --version
+    python budget/tests.py
+deps =
+    -rdev-requirements.txt
+    -rrequirements.txt
+
+install_command = pip install --pre {opts} {packages}
+
+[testenv:docs]
```

Comment:
The use of tox to build docs surprised me at first but why not, it could improve doc quality after all :-)